tasks/install_targz.yml

```diff
@@ -15,6 +15,7 @@
   file:
     path: "{{ nrinfragent_tarball_download_path }}"
     state: directory
+    mode: 0666
```

Comment:
is it not too open?

Reply:
I've change this to 0600 should be better now